docs/guides/external-node/08_pruning.md

```diff
@@ -0,0 +1,38 @@
+# Pruning
+
+It is possible to configure Node to periodically remove all data from batches older than a threshold, both from Postgres
+and from tree.
+
+> [!NOTE]
+>
+> If you need a node with data retention period of up to a few days, please set up a node from a snapshot (see previous
+> chapter) and wait for it to have enough data. Pruning an archival node can take unpractical amount of time. In the
+> future we will be offering pre-pruned DB snapshots with a few months of data.
+
+You can enable pruning by setting
+
+```
+EN_PRUNING_ENABLED=true
+```
+
+By default, it will keep history for 7 days. You can configure retention period using:
+
+```
+EN_PRUNING_DATA_RETENTION_SEC: '259200' // 3 days
+```
+
+The data retention can be set to any value, but values under 21h will as the batch can only be pruned as soon as it has
+been executed on Ethereum.
```

Comment:
- As mentioned in another comment, IMO the overall concept of unchangeable pruning restrictions should be introduced earlier.
- "but values under 21h will" – did you a part of the sentence here? It looks unfinished.
- IMO, worth clarifying that the execution latency depends on the network and 21h is the mainnet value.

Reply:
I've removed "be ignored" from that sentence by mistake. I've added that 21h is for mainnet.